addons/survey/views/survey_templates.xml

```diff
@@ -183,10 +183,10 @@
         </div>
 
         <form role="form" method="post" class="js_surveyform" t-att-name="'%s_%s' % (survey.id, page.id)"
-                t-att-action="'/survey/fill/%s/%s' % (survey.id, token)"
-                t-att-data-prefill="'/survey/prefill/%s/%s?page_id=%s' % (survey.id, token, page.id)"
-                t-att-data-validate="'/survey/validate/%s/%s' % (survey.id, token)"
-                t-att-data-submit="'/survey/submit/%s/%s' % (survey.id, token)">
+                t-att-action="'/survey/fill/%s/%s' % (survey.access_token, token)"
```

Comment:
@tde-banana-odoo Same here the second "token" parameter is a little confusing. Would probably be better to name it "answer_token" as well.